lib/smart_answer_flows/marriage-abroad/outcomes/countries/tunisia/ceremony_country/partner_british/_opposite_sex.govspeak.erb

```diff
@@ -53,6 +53,6 @@ The consulate will display your notice of marriage publicly for 7 days.
 
 You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for Tunisia](/government/publications/tunisia-consular-fees).
 
-<%=  render partial: 'how_to_pay.govspeak.erb', locals: {calculator: calculator}%>
+You can pay at your appointment using a credit or debit card or cash in the local currency. 
```

Comment:
do we want to turn this in to a hard coded string or is it worth creating a partial for this and changing [`payment_partial_name` for Tunisia](https://github.com/alphagov/smart-answers/blob/master/lib/data/marriage_abroad_services.yml#L668)?

Reply:
It's probably worth creating a partial. Doing that now.